Epicure/src/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,28 @@
+import NevigationBar from "../NevigationBar/NevigationBar.tsx";
+import UserIcon from "../../assets/Icons/Person.png";
+import MenuIcon from "../../assets/Icons/Menu.png";
+import ExploreIcon from "../../assets/Icons/Explore.png";
+import OrderIcon from "../../assets/Icons/Order.png";
+import HeaderImage from "../../assets/Images/Header.png";
+import {
+  HeaderContainer,
+  HeaderImg,
+} from "./HeroStyles.ts";
+import SearchBar from "../SearchBar/SearchBar.tsx";
+
+function Hero() {
+  const icons = [MenuIcon, ExploreIcon, UserIcon, OrderIcon];
+  const placeholder = "Search for restaurant cuisine, chef";
```

Comment:
lets make a separate file which will contain all the texts in the website
you can name it as resources.ts
which will contain an object with all the necessary texts and their locations, for example:
```
export const RESOURCES =  {
    RESTAURANTS_TITLE = "example",
    RESTAURANTS_TITLE_PLACEHOLDER = "example2",
    ...,
    ...,
    ...,
}
```